src/views/f2f/prove-identity-post-office.html

```diff
@@ -4,7 +4,6 @@
 {% set hmpoTitle = landingPage.pageTitle %}
 {% set hmpoPageKey = "landingPage" %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
-{% from "govuk/components/header/macro.njk" import govukHeader %}
```

Comment:
How come we removed this?

Reply:
This interacts with the cookie banner in some way and prevents it from rendering